src/components/Activity.js

```diff
@@ -1,20 +1,20 @@
 import React, { useEffect, useState } from "react";
 import Typography from "@material-ui/core/Typography";
 import MenuItem from "@material-ui/core/MenuItem";
-import Chip from "@material-ui/core/Chip";
+// import Chip from "@material-ui/core/Chip";
```

Comment:
Let's just delete all these unused lines of code in `Activity.js`

- [ ] delete unused lines

Reply:
Those look meaningful to me, like they're supposed to be able to toggled back on quickly, so let's leave those for now.